doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +169,113 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
+and recommend using this for setting up your stack.
+
+.. note::
+
+  See the `Heptio's note on security <http://blog.kubernetes.io/2016/08/security-best-practices-kubernetes-deployment.html>`_
+  for information about keeping your Kubernetes infrastruture secure.
+
+1. Follow Step 1 of the `Heptio guide`_.
+
+   This sets up your Amazon account with the credentials needed to run Kubernetes.
+
+   .. note::
+
+      Make sure that you keep the file downloaded when you create the SSH
+      key. This will be needed later to allow ``kubectl`` to interact with
+      your Kubernetes cluster.
+
+   .. note::
+
+      You may find it helpful to "pin" the services we'll be using to your AWS
+      navbar. This makes it easier to navigate in subsequent sessions.
+      Click the "pin" icon at the top, then drag ``CloudFormation`` and
+      ``EC2`` into your navbar.
+
+2. Follow Step 2 of the `Heptio guide`_.
+
+   **Select template option**: In this section, Heptio allows you
+   to click one of two buttons, each runs several commands on AWS
+   that set up a "template" of necessary Kubernetes resources.
+   We recommend ``Option 1``, which will create a new set of resources
+   on AWS to run Kubernetes.
+
+   After clicking the ``Option 1`` button, you'll be taken to an AWS page with a field already
+   chosen under "Choose a template". Simply hit "Next".
+
+   **Enter AWS instance information (page 1)**: On this page you'll need to
+   fill in the following required fields:
+
+   * ``Stack Name`` can be anything you like.
+   * ``Availability Zone`` is related to the location of the AWS
+     resources. Choose an AWS location close to your physical location or
+     any other desired AWS location.
+   * ``Admin Ingress Location`` defines the locations from which you
+     can access this cluster as an administrator. Enter ``0.0.0.0/0``
+     for the most permissive approach.
+   * ``SSH Key`` is a dropdown list of keys attached to your account.
+     The one you created in Step 1 should be listed here. This will allow
+     you to SSH into the machines if you desire.
+   * ``Node Capacity`` defines the number of machines you've got available.
+     This will depend on the ``Instance Type`` that you choose. E.g., if you
+     want each user to have 2GB and you expect 10 users, choose a combination
+     of ``Instance Type`` and ``Node Capacity`` that meets this requirement.
+   * ``Instance Type`` defines what kind of machine you're requesting. See
+     this `list of instance types with Amazon <https://aws.amazon.com/ec2/instance-types/>`_
+     as well as this list of `pricing for each instance type <https://aws.amazon.com/ec2/pricing/on-demand/>`_.
+   * ``Disk Size`` corresponds to the hard disk for each node. If
+     you need users to keep lots of large files that persist over time,
+     this should be larger.
+   * ``Instance Type (Bastion Host)`` corresponds to a "manager"
```

Comment:
It isn't a 'manager' instance - it just is an easy place to ssh in and connect to the kubernetes master (which is the 'manager' type instance and different from bastion). http://docs.aws.amazon.com/quickstart/latest/linux-bastion/architecture.html has detailed description.